src/libs/actions/Report.js

```diff
@@ -1531,6 +1534,30 @@ function createPolicyRoom(policyID, reportName, visibility) {
         .finally(() => Onyx.set(ONYXKEYS.IS_LOADING_CREATE_POLICY_ROOM, false));
 }
 
+/**
+ * Renames a user created Policy Room.
+ * @param {String} reportID
+ * @param {String} reportName
+ */
+function renameReport(reportID, reportName) {
+    Onyx.set(ONYXKEYS.IS_LOADING_RENAME_POLICY_ROOM, true);
+    API.RenameReport({reportID, reportName})
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                Growl.error(response.message);
+                return;
+            }
+            Growl.success(Localize.translateLocal('newRoomPage.policyRoomRenamed'));
+
+            // Optimistically update the report name
```

Comment:
NAB That's not really an optimistic update since you are updating after receiving the response. I think it's fine to wait for the response to update the name since it's so quick.

Reply:
Good point! I'll update the comment.